Mathlib/CategoryTheory/Limits/Preserves/Shapes/Zero.lean

```diff
@@ -180,4 +180,23 @@ def preservesInitialObjectOfPreservesZeroMorphisms [PreservesZeroMorphisms F] :
 
 end ZeroObject
 
+section
+
+variable (J : Type*) [Category J] [HasZeroObject D] [HasZeroMorphisms D]
+  (G : C ⥤ D) (hG : IsZero G)
+
+/-- A zero functor preserves limits. -/
+def preservesLimitsOfShapeOfIsZero : PreservesLimitsOfShape J G where
+  preservesLimit {K} := ⟨fun hc => by
+    rw [Functor.isZero_iff] at hG
+    exact IsLimit.ofIsZero _ ((K ⋙ G).isZero (fun X ↦ hG _)) (hG _)⟩
+
+/-- A zero functor preserves colimits. -/
+def preservesColimitsOfShapeOfIsZero : PreservesColimitsOfShape J G where
+  preservesColimit {K} := ⟨fun hc => by
+    rw [Functor.isZero_iff] at hG
+    exact IsColimit.ofIsZero _ ((K ⋙ G).isZero (fun X ↦ hG _)) (hG _)⟩
```

Comment:
Can you upgrade them to `PreservesLimitsOfSize`?

Reply:
Thanks for the suggestion.